.github/workflows/ci.yml

```
@@ -0,0 +1,17 @@
+name: CI
+
+on: [push]
```

```suggestion
on: [push, pull_request]
```
Do we want this so that we can see how it behaves on your PR?